### ISSUE TITLE
Citation

### ISSUE THREAD

**Issue by ju-resplande** [pemistahl/lingua-py#230]
Hi there,

I am using your repo for my master's thesis. How I can cite it?

Thanks! :)

**Comment by pemistahl** [comment]
Hi Juliana, you can cite it like this:

Peter M. Stahl: _Lingua - An accurate natural language detection library for short and mixed-language text_ (https://github.com/pemistahl/lingua-py)

I'm curious to read your master's thesis. What is it about? :)

**Comment by Stormur** [comment]
Hi!

Are there no other publications/documentation files available?

**Comment by ju-resplande** [comment]
> Hi Juliana, you can cite it like this:
> 
> Peter M. Stahl: _Lingua - An accurate natural language detection library for short and mixed-language text_ (https://github.com/pemistahl/lingua-py)
> 
> I'm curious to read your master's thesis. What is it about? :)

Hi! It is about fact-checking.  Some datasets were multilingual and I wanted to get only Portuguese language examples. Still I checked manually some examples because Spanish and Portuguese are really similar languages, specially in few-words scenerios like tweets, one word or even a letter make the difference to separate those languages. BTW, I cited on two papers: in my master thesis and one paper that published in a workshop 😄 .
